### DISCUSSION TITLE
how to I use chatgpt-wrapper as chat

### DISCUSSION THREAD

**Discussion by robertoberto** [llm-workflow-engine/llm-workflow-engine#260] [Q&A]
chatgpt doesnt remember my conversion history

(0.9/1/0.6/0/4000/23)
roberto@gpt-3.5-turbo 6> /new

 • New conversation started.                                                                                                                                

(0.9/1/0.6/0/4000/0)
roberto@gpt-3.5-turbo 7> /ask hi my name is elder

Hello Elder, how may I assist you today?

(0.9/1/0.6/0/4000/23)
roberto@gpt-3.5-turbo 8> /ask what is my name

I'm sorry, but I don't know your name. You can tell me your name and I'll address you by it.

backend: chatgpt-api                                                                                                                                       
 browser:                                                                                                                                                   
   debug: false                                                                                                                                             
   provider: firefox                                                                                                                                        
 chat:                                                                                                                                                      
   log:                                                                                                                                                     
     enabled: false                                                                                                                                         
     filepath: chatgpt.log                                                                                                                                  
   model: default                                                                                                                                           
   model_customizations:                                                                                                                                    
     frequency_penalty: 0                                                                                                                                   
     max_submission_tokens: 4000                                                                                                                            
     presence_penalty: 0.6                                                                                                                                  
     system_message:                                                                                                                                        
       default: You are a helpful assistant.                                                                                                                
       programmer: You are an expert programmer, who responds to questions with brief                                                                       
         examples in code.                                                                                                                                  
     temperature: 0.9                                                                                                                                       
     top_p: 1                                                                                                                                               
   streaming: true

**Comment by thehunmonkgroup** [comment]
Cannot reproduce:

```
(0.5/1/0.6/0/4000/0)
hunmonk@gpt-3.5-turbo 1> /new

 • New conversation started.                                                                                                           

(0.5/1/0.6/0/4000/0)
hunmonk@gpt-3.5-turbo 2> /ask hi my name is chad

Hello Chad! How may I assist you today?

(0.5/1/0.6/0/4000/39)
hunmonk@gpt-3.5-turbo 3> /ask what is my name?

Your name is Chad.

(0.5/1/0.6/0/4000/59)
hunmonk@gpt-3.5-turbo 4>
```

I would guess you ran into a transient error w/ OpenAI's servers, and would recommend waiting and trying later.

If it persists, I don't currently have any idea what in this project would cause that. Probably would require you to hack the code and put in some debugging statements.

**Comment by robertoberto** [comment]
Hi

Maybe I am using 
backend: chatgpt-api

and you  maybe browser based

I setup database properly, database file is being bigger

```

roberto@gpt-3.5-turbo 4> /new

 • New conversation started.                                                                                                                                

(0.9/1/0.6/0/4000/0)
roberto@gpt-3.5-turbo 5> /ask my name is roberto

Hello Roberto! How can I assist you today?

(0.9/1/0.6/0/4000/23)
roberto@gpt-3.5-turbo 6> /ask whats my name

I'm sorry, but as an AI language model, I don't have access to personal information such as your name. Can you please tell me how can I assist you today?

(0.9/1/0.6/0/4000/22)

```

**Comment by thehunmonkgroup** [comment]
No, my test was with the `chatgpt-api` backend as well.

**Comment by thehunmonkgroup** [comment]
This issue should be fixed in the latest release.
